backend/src/schema/resolvers/rooms.ts

```diff
@@ -0,0 +1,56 @@
+import { v4 as uuid } from 'uuid'
+import { neo4jgraphql } from 'neo4j-graphql-js'
+import Resolver from './helpers/Resolver'
+
+export default {
+  Query: {
+    Room:  async (object, params, context, resolveInfo) => {
+      if (!params.filter) params.filter = {}
+      params.filter.users_some = {
+        id: context.user.id,
+      }
+      return neo4jgraphql(object, params, context, resolveInfo)
+    },
+  },
+  Mutation: {
+    CreateRoom: async (_parent, params, context, _resolveInfo) => {
+      const { userId } = params
+      const { user: { id: currentUserId } } = context
+      const roomId = uuid()
```

Comment:
This has collision problems.

Please use the database instead:
```
apoc.create.uuid()
```
Ref: https://neo4j.com/labs/apoc/4.1/graph-updates/uuid/

You would actually not require an extra id(the uuid), since every node has a `native ID` which has the disadvantage that it could be reused once the node was deleted. Therefore I suggest to use the uuid, but please with the database.

Further read:
https://stackoverflow.com/questions/60748609/neo4j-generate-unique-id-for-existing-node